### PR TITLE
Auth test rewrite

### PR DESCRIPTION
**Pull request by lavanyareddyreddy**
#### What tests were added (if relevant), and do all tests (rspec, karma, and integration CALS tests) pass?
-no

#### Which User story does this relate to?
-#146720813

#### Brief feature description
-currently the Ruby test coverage is 88%, and bring code test coverage to 100%

#### Does this PR block/effect other stories?
-no

#### Do any commands needs to be run in order to run the project with this PR?
-rspec spec

#### Code coverage percent diff as a result of this PR?
-n/a

#### JS errors diff as a result of this PR?
-n/a

### CODE REVIEW

**Comment by ratneshraval** [comment]
@lavanyareddyreddy Take a look at this https://relishapp.com/rspec/rspec-rails/v/3-6/docs/controller-specs/anonymous-controller

File: spec/controllers/cals_base_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+describe CalsBaseController do
+
+  before(:each) do
+    byebug
+    allow(controller). to receive_messages(Cwds::Authentication.token_validation => 'abcd')
+    #allow_any_instance_of(CalsBaseController).to receive(Cwds::Authentication.token_validation).and_return('abcd')
+  end
+
+  it 'should store token if session token is empty' do
+    byebug
```
Comment (lauherkcambria) [inline spec/controllers/cals_base_controller_spec.rb]: take out byebug

File: spec/controllers/cals_base_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+    allow(controller). to receive_messages(Cwds::Authentication.token_validation => 'abcd')
+
+    expect(session['token']).to eq('abcd')
+  end
+
+
+  it 'should redirect to a URL if session token is present' do
+
+    byebug
```
Comment (lauherkcambria) [inline spec/controllers/cals_base_controller_spec.rb]: remove byebug

Comment (ratneshraval) [inline spec/controllers/cals_base_controller_spec.rb]: @lauherkcambria you are working on a deleted file.

Comment (lauherkcambria) [inline spec/controllers/cals_base_controller_spec.rb]: yeah, that's why i approved it haha

**Pull request merged**